app/main/views/frameworks.py

```diff
@@ -1213,3 +1217,20 @@ def view_contract_variation(framework_slug, variation_slug):
 @login_required
 def opportunities_dashboard_deprecated(framework_slug):
     return redirect(url_for('external.opportunities_dashboard', framework_slug=framework_slug), code=301)
+
+
+@main.route('/frameworks/<framework_slug>/legal-authority', methods=['GET', 'POST'])
+@login_required
+def legal_authority(framework_slug):
```

Comment:
We should have a check at the beginning of the view to see if the supplier has made an application or not. They can't start the signing process if their `supplierFramework` object does not have `onFramework: true`.